library/pod-security-policy/users/constraint.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sPSPAllowedUsers
+metadata:
+  name: psp-pods-allowed-user-ranges
+spec:
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
+        namespace: ["default"]
```

Comment:
do we want to have the default namespace here? in other examples, we don't specify a namespace

Reply:
+1 namespace is superfluous hear.

It also should be at a lower indentation level if we want to keep it.

Reply:
Thanks, removed it.